src/03-control-flow.js

```diff
@@ -109,7 +180,25 @@ function aroundTheWorldGreeting(language) {}
  *  aroundTheWorldGreetingWithSwitch();
  *  //> "Hello World"
  */
-function aroundTheWorldGreetingWithSwitch(language) {}
+function aroundTheWorldGreetingWithSwitch(language) {
+ switch (language) {
+  case 'english':
+    return 'Hello World';
+    break;
```

Comment:
Since the `return` stops your code from running, you don't need to include a `break`. The returns acts like the `break`